src/styles/stars.scss

```diff
@@ -0,0 +1,28 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+
+  &__star {
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    width: $star-size;
```

Comment:
Ensure that the variable `$star-size` is defined in your SCSS files. If it's not defined, it will cause an error during compilation.